boards/shields/npm1100_ek/Kconfig.defconfig

```diff
@@ -1,9 +0,0 @@
-# Copyright (c) 2023 Nordic Semiconductor ASA
-# SPDX-License-Identifier: Apache-2.0
-
-if SHIELD_NPM1100_EK
-
-config REGULATOR
```

Comment:
if it's not too much to ask, could you add CONFIG_REGULATOR=y to https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/shields/npm6001_ek/prj.conf

Reply:
Sure, thanks for the heads up.